core/templates/pages/topic-editor-page/topic-editor-page.component.ts

```diff
@@ -111,6 +111,19 @@ angular.module('oppia').directive('topicEditorPage', [
           ctrl.selectQuestionsTab = function() {
             TopicEditorRoutingService.navigateToQuestionsTab();
           };
+          ctrl.getNavbarText = function() {
+            if (TopicEditorStateService.hasLoadedTopic()) {
+              const activeTab = ctrl.getActiveTabName();
+              if (activeTab === 'main') {
+                return (
+                  `Topic Editor: ${ctrl.topic.getName()}` +
```

Comment:
Sorry for harping about this again but found a nit: In the screenshot, it looks like you have a space after the topic name and before `v(..)`. If so, please add it -- it's fine to add in your next PR.

Reply:
Done.